views/ThingSearchPage.qml

```diff
@@ -18,6 +18,17 @@ ColumnLayout
         Layout.topMargin: (parent.height / 2) - (height / 2)
     }
 
+    Cura.SecondaryButton
```

Comment:
This button needs some more top margin (check the thing details page 'go back' button for a good height).